README.md

```diff
@@ -127,13 +127,12 @@ This extension augments VS Code in the following ways:
 Open the Command Palette (`F1` or ⇧⌃P on Windows/Linux or ⇧⌘P on macOS) and type
 in one of the following commands:
 
-| Command                    | Description                                                                                                                 |
-| -------------------------- | --------------------------------------------------------------------------------------------------------------------------- |
-| `DVC: Get Started`         | Open the extension's walkthrough. Which details all of the current features and provides links to extra learning resources. |
```

Comment:
[F] We want people to access this and `DVC: Setup The Workspace` via the setup page so I have removed both the entries.